editable-mask-layer/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "carto-deckgl-example-editable-mask-layer",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "dev-local": "vite --config ../vite.config.local.mjs",
+    "build": "vite build",
+    "preview": "vite preview",
+    "link-deck": "yarn link @deck.gl/core && yarn link @deck.gl/layers && yarn link @deck.gl/geo-layers && yarn link @deck.gl/mesh-layers && yarn link @deck.gl/aggregation-layers && yarn link @deck.gl/carto && yarn link @deck.gl/extensions",
+    "unlink-deck": "yarn unlink @deck.gl/core && yarn link @deck.gl/layers && yarn link @deck.gl/geo-layers && yarn link @deck.gl/mesh-layers && yarn link @deck.gl/aggregation-layers && yarn link @deck.gl/carto && yarn link @deck.gl/extensions"
+  },
+  "devDependencies": {
+    "vite": "^4.5.0"
+  },
+  "dependencies": {
+    "@carto/api-client": "^0.3.0",
+    "@deck.gl-community/editable-layers": "9.1.0-beta.4",
+    "@deck.gl/aggregation-layers": "9.1.8",
+    "@deck.gl/carto": "9.1.8",
+    "@deck.gl/core": "9.1.8",
+    "@deck.gl/extensions": "9.1.8",
+    "@deck.gl/geo-layers": "9.1.8",
+    "@deck.gl/layers": "9.1.8",
+    "@deck.gl/mesh-layers": "9.1.8",
+    "@turf/helpers": "^7.2.0",
+    "lodash": "^4.17.21",
+    "maplibre-gl": "^3.5.2"
+  }
```

Comment:
I think we might be missing some dependencies that would allow this to work in a standalone folder and/or CodeSandbox, if I do this locally...

```bash
cd editable-mask-layer
touch yarn.lock
yarn
yarn dev
```

... I'm seeing errors about missing `@luma.gl/engine`, `@loaders.gl/core`,  and a few other packages. Should `typescript` also be in devDependencies? I see this looks similar to some other examples though, maybe I've misunderstood how the examples work. 🤔

Reply:
The errors about luma are usually conflicts with previously existing or conflicting deck.gl versions. I have used pnpm to try these examples. Maybe these do not work fresh out of the repo with yarn specifically ?

Let me try again and I'll confirm

Reply:
Oops, I chatted with @srtena about this and it turns out it's Yarn. With npm v7+ any peerDependencies of dependencies are installed automatically, with Yarn and pnpm they are not. pnpm has an option to turn that on. 

In any case, seems like it's unrelated to this PR, and this example matches the rest, so LGTM!